frontend/src/pages/user-profile/components/user-profile-sidebar/user-profile-sidebar.tsx

```diff
@@ -0,0 +1,29 @@
+import { useAppSelector } from '#libs/hooks/hooks.js';
+
+import styles from './styles.module.scss';
+
+const UserProfileSidebar: React.FC = () => {
+  const { authenticatedUser } = useAppSelector(({ auth }) => ({
+    authenticatedUser: auth.authenticatedUser,
+  }));
+
+  return (
+    <div className={styles['container']}>
+      <div className={styles['header']}>Profile</div>
+      <div className={styles['user-info']}>
+        <div className={styles['avatar-container']}>
+          <img
+            src="images/avatar.png"
+            alt="avatar"
+            className={styles['avatar']}
+          />
+        </div>
+        <span className={styles['user-name']}>
+          {authenticatedUser?.fullName}
```

Comment:
We will always have a user here, cast the type

Reply:
done